indico/web/forms/fields.py

```diff
@@ -846,12 +846,18 @@ def __init__(self, *args, **kwargs):
         self.protected_object = kwargs.pop('protected_object')(kwargs['_form'])
         get_acl_message_url = kwargs.pop('acl_message_url', None)
         self.acl_message_url = get_acl_message_url(kwargs['_form']) if get_acl_message_url else None
+        self.inherits_protection = kwargs.pop('inherits_protection', lambda form: False)(kwargs['_form'])
```

Comment:
Not a fan of the naming. What you actually have here is something like `can_inherit_protection`